packages/hdwallet-native/src/bitcoin.ts

```diff
@@ -35,6 +35,7 @@ type InputData = UtxoData | ScriptData | BchInputData;
 
 export function MixinNativeBTCWalletInfo<TBase extends core.Constructor<core.HDWalletInfo>>(Base: TBase) {
   // eslint-disable-next-line @typescript-eslint/no-shadow
+  // @ts-ignore tsc cache is drunk, btcIsSameAccount has been removed, TODO(gomes): rm ts-ignore
```

Comment:
dont' forget

Reply:
27297629e92051d6442f056b0da56dd8a915a0f2 - keeping opened in case it comes back biting us to nuke again